modules/xmpp/BreakoutRooms.js

```diff
@@ -131,6 +132,47 @@ export default class BreakoutRooms {
         this._sendMessage(message);
     }
 
+    /**
+     * Sends a message to a breakout room.
+     *
+     * @param {string} roomJid - JID of the room to send the message to.
+     * @param {string} textContent - The message content.
+     */
+    sendMessageToRoom(roomJid, textContent) {
```

Comment:
These new functions should check for the broadcast capability I think.

Reply:
added check for feature, using the already existing `isFeatureSupported` function